functions/slack/main.py

```diff
@@ -89,6 +90,7 @@ def make_search_request(query):
 
 
 # [START functions_slack_search]
+@functions_framework.http
```

Comment:
Sorry I'm not too informed about this decorator, is this for good practice? If this is a recommended approach, and this is going to show up on the Sample Browser as well, I think it might be good to be able to point users towards some sort of documentation that explains why using this might be beneficial!

Reply:
I don't think it was documented; This PR https://github.com/GoogleCloudPlatform/functions-framework-nodejs/pull/347 is the most detailed bit I could find. Thoughts on whether we should point to it?